scene/gui/line_edit.cpp

```diff
@@ -203,29 +203,30 @@ void LineEdit::_gui_input(Ref<InputEvent> p_event) {
 
 				case (KEY_X): { // CUT
 
-					if (editable) {
+					if (editable && is_shortcut_keys_enabled()) {
```

Comment:
Instead of adding this check in all cases, maybe simply put it in the outer `if`?